admin/timescale/updates/2022-09-21-phase-out-mbid-mapping-metadata.sql

```diff
@@ -0,0 +1,32 @@
+BEGIN;
```

Comment:
Do you have this file for the sole purpose of people updating their dev envs? Or does it serve another purpose?

Reply:
Yeah, mostly for dev envs because this table is now needed to serve listens as well.